wewatch-client/src/views/swiping/swiping-container.js

```diff
@@ -21,7 +21,12 @@ const SwipingContainer = (props) => {
 
   const [index, setIndex]  = useState(0);
   const [title, setTitle] = useState(topTenMovies[index].title);
-  const [imageURL, setImageUrl] = useState(topTenMovies[index].image)
+  const [imageURL, setImageUrl] = useState(topTenMovies[index].image);
+  const [year, setYear] = useState(topTenMovies[index].released);
+  const [lengthOfMovie, setLengthOfMovie] = useState(topTenMovies[index].duration);
+  const [rating, setRating] = useState(topTenMovies[index].rating);
+  const [genre, setGenre] = useState(topTenMovies[index].type);
```

Comment:
Maybe you can just change this variable name to typeOfMovie and name the function setTypeOfMovie to coincide with the movie type

Reply:
Yeah, that makes more sense, changes pushed!